packages/botonic-nlu/src/ner.js

```diff
@@ -0,0 +1,39 @@
+import { default as nlp } from 'compromise'
+import { default as compromisePlugin } from 'compromise-plugin'
+import { DEFAULT_ENTITIES } from './constants'
+
+function listContainEntity(obj, list) {
+  for (let item of list) {
+    if (obj.type == item.type && obj.value == item.value) {
+      return true
+    }
+  }
+  return false
+}
+
+export function getEntities(input, devEntities) {
```

Comment:
I suggest creating a class DevEntities (maybe with another) name with its fields (tagList, tags, words,...) declared and briefly documented

Reply:
i would review this part now when I will try to add more features to the entity system 👍